src/widgets/indexer-manager/IndexerManagerTile.tsx

```diff
@@ -0,0 +1,101 @@
+import { Card, Center, Flex, Group, Stack, Text } from '@mantine/core';
+import { IconReportSearch, IconUnlink } from '@tabler/icons-react';
+import { useEffect, useState } from 'react';
+
+import { defineWidget } from '../helper';
+import { IWidget } from '../widgets';
+
+const definition = defineWidget({
+  id: 'indexer-manager',
+  icon: IconReportSearch,
+  options: {
+    url: {
+      type: 'text',
+      defaultValue: '',
+    },
+    apiKey: {
+      type: 'text',
+      defaultValue: '',
+    },
+  },
+  gridstack: {
+    minWidth: 1,
+    minHeight: 1,
+    maxWidth: 2,
+    maxHeight: 2,
+  },
+  component: IndexerManagerWidgetTile,
+});
+
+export type IIndexerManagerWidget = IWidget<(typeof definition)['id'], typeof definition>;
+
+interface IndexerManagerWidgetProps {
+  widget: IIndexerManagerWidget;
+}
+
+function IndexerManagerWidgetTile({ widget }: IndexerManagerWidgetProps) {
+  const [indexerData, setIndexerData] = useState([]);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    const fetchIndexerStatus = async () => {
+      try {
+        const response = await fetch(`${widget.properties.url}/api/v1/indexer`, {
```

Comment:
Do not fetch from the client. Additionally, it's a bad idea to use ``useEffect()`` for fetching. Make use of tRPC!